platform-core-module/src/main/java/com/softicar/platform/core/module/file/stored/content/StoredFileContentMigrationExecutor.java

```diff
@@ -0,0 +1,107 @@
+package com.softicar.platform.core.module.file.stored.content;
+
+import com.softicar.platform.common.core.exception.ExceptionsCollector;
+import com.softicar.platform.common.core.logging.Log;
+import com.softicar.platform.common.io.StreamUtils;
+import com.softicar.platform.core.module.file.stored.content.store.StoredFileSmbContentStore;
+import com.softicar.platform.core.module.file.stored.hash.AGStoredFileSha1;
+import com.softicar.platform.core.module.file.stored.server.AGStoredFileServer;
+import com.softicar.platform.core.module.module.instance.AGCoreModuleInstance;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Collection;
+import java.util.stream.Collectors;
+
+public class StoredFileContentMigrationExecutor {
+
+	private static final String INCOMING_FILENAME_SUFFIX = ".part";
+	private final ExceptionsCollector exceptionsCollector;
+	private StoredFileSmbContentStore primaryStore;
+	private Collection<StoredFileSmbContentStore> secondaryStores;
+
+	public StoredFileContentMigrationExecutor() {
+
+		this.exceptionsCollector = new ExceptionsCollector();
+	}
+
+	public void execute() {
+
+		AGStoredFileServer primaryFileServer = AGCoreModuleInstance.getInstance().getPrimaryFileServer();
+		if (primaryFileServer != null && primaryFileServer.isActive()) {
+			this.primaryStore = new StoredFileSmbContentStore(primaryFileServer);
```

Comment:
For the sake of eymmetry:

I suggest to either add a `File` infix here as well (so this would become `primaryFileStore`), **or** to remove the infixin the line above (so `primaryFileServer` would become `primaryServer`).

**Alternatively**, we could create a private method `loadPrimaryStore` (just like the existing `loadSecondaryStores`) in this class.